ditto/src/main/java/eu/xenit/testing/ditto/api/AlfrescoBootstrapper.java

```diff
@@ -108,12 +183,81 @@ T bootstrap(T builder) {
         };
     }
 
+    private Consumer<NodeCustomizer> sitesNode(String name, Consumer<NodeCustomizer> callback) {
+        return node -> {
+
+            node.name(name);
+            node.qname(Site.createQName(name.toLowerCase().replace(" ", "_")));
```

Comment:
can you abstract this repetition out?

Reply:
I don't really understand what you're trying to say? What's the repetition?

Reply:
Got it, fixed.